tests/bundle/Functional/xsd/Version.xsd

```diff
@@ -106,6 +106,15 @@
             </xsd:simpleContent>
           </xsd:complexType>
         </xsd:element>
+        <xsd:element name="Thumbnail">
```

Comment:
cc @ezsystems/documentation-team - REST spec for Version object to be updated - we have a new property.